Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -248,12 +250,18 @@ public enum SlipControlType
         float DebugSpeed = 1; // Used for debugging adhesion coefficient
 
         // parameters for Track Sander based upon compressor air and abrasive table for 1/2" sand blasting nozzle @ 50psi
-        public float MaxTrackSandBoxCapacityM3 = Me3.FromFt3(40.0f);  // Capacity of sandbox - assume 40.0 cu ft
-        public float CurrentTrackSandBoxCapacityM3; 
-        public float TrackSanderAirComsumptionM3pS = Me3.FromFt3(195.0f) / 60.0f;  // Default value - cubic feet per min (CFM) 195 ft3/m
-        public float TrackSanderAirPressurePSI = 50.0f;
-        public float TrackSanderSandConsumptionM3pS = Me3.FromFt3(11.6f) / 3600.0f; // Default value - 11.6 ft3/h
+        public float MaxTrackSandBoxCapacityM3; // Capacity of sandbox
+        public float MaxTrackSanderAirComsumptionForwardM3pS;
+        public float MaxTrackSanderAirComsumptionReverseM3pS = 0;
+        public float MaxTrackSanderSandConsumptionForwardM3pS;
+        public float CurrentTrackSanderAirConsumptionM3pS;
+        public float CurrentTrackSanderSandConsumptionM3pS;
+        public float CurrentTrackSandBoxCapacityM3;
```

Comment:
```suggestion
        public float CurrentTrackSandBoxContentM3;
```
Just a suggestion. To me "Capacity" means "maximum quantity it can contain"

Reply:
I understand your thought, however I didn't take it the same way as Max and Current differentiate the capacities.

Also I don't want to change it as I suspect that it will "null and void" your approval.